README.md

```diff
@@ -37,15 +37,30 @@ Note, however, that DNF5 cannot yet interface with the following programming lan
 Installing DNF5
 ---------------
 
-To install the DNF5 package manager, use either of the following commands, depending on your Fedora release:
+DNF5 is available since Fedora 38. To install the DNF5 package manager, use either of the following commands, depending on your Fedora release:
 
 | Fedora release | Command |
 | --------------------------- | ------- |
-| 36 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-unstable && sudo dnf install dnf5`[^copr] |
-| 37 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-unstable && sudo dnf install dnf5`[^copr] |
+| 36 | `sudo dnf copr enable rpmsoftwaremanagement/dnf-nightly && sudo dnf install dnf5`[^copr] |
+| 37 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-nightly && sudo dnf install dnf5`[^copr] |
 | 38                 | `sudo dnf install dnf5` |
+| 39                 | `sudo dnf install dnf5` |
```

Comment:
I would remove a line for Fedora 36 and added a line for Fedora 40.

Reply:
oh, right, I will change that

Reply:
I actually changed it to `38 or newer`, what do you think?